morl_baselines/multi_policy/envelope/envelope.py

```diff
@@ -497,6 +506,16 @@ def train(
                 self.update()
 
             if eval_env is not None and self.log and self.global_step % eval_freq == 0:
```

Comment:
I think we need to change this to log the front every "iteration", instead of every eval_freq, similarly as in GPI-PD